app/services/hawkular_proxy_service.rb

```diff
@@ -1,6 +1,15 @@
 class HawkularProxyService
   include UiServiceMixin
 
+  MAX_LEN = 25
+  SPECIAL_CASES = {
```

Comment:
@yaacov I think should be more meaningful of "special cases" (think of other engineers that are not familiar with what you're doing... special cases of what?)